ui/src/client/index.js

```diff
@@ -3,17 +3,26 @@ import './styles/styles.scss';
 import { ApolloClient } from 'apollo-client';
 import { ApolloProvider } from 'react-apollo';
 //import gql from 'graphql-tag';
-import { HttpLink } from 'apollo-link-http';
+import { createHttpLink } from 'apollo-link-http';
 import { InMemoryCache } from 'apollo-cache-inmemory';
 
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { AppContainer } from 'react-hot-loader';
 import App from './components/App';
 
+const link = createHttpLink({
+  uri: 'https://localhost:3001/graphql',
```

Comment:
Should this just be `/graphql` since the UI server is proxying this? Also will help when we deploy since the host:port combo won't be same.